app/controllers/Application.scala

```diff
@@ -154,7 +154,11 @@ object Application extends Controller {
   def npmPackageVersions(packageName: String) = Action.async {
     val packageVersionsFuture = Cache.getAs[Seq[String]](s"npm-versions-$packageName").fold {
       npm.info(packageName).map { json =>
-        val versions = (json \ "versions" \\ "version").map(_.as[String])
+        val versions = (json \ "versions" \\ "version")
+          .map(_.as[String])
+          .map(WebJarVersion(_))
+          .sorted(WebJarVersion.WebJarVersionOrdering.reverse)
+          .map(_.number)
```

Comment:
Not sure if this is the best way to do it. Since WebJarVersion has the optional numFiles argument, importing the implicit JS converter wasn't working -- admittedly this didn't make sense to me, as it is an optional field, but don't really have the time to look at it too closely right now.